web/src/js/script.js

```diff
@@ -79,58 +86,90 @@ function ConfigItem(name, factory, apply, fetch, help) {
+}
+
 var CONFIG_ITEMS = [
-    new ConfigItem("deviceName", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mdnsName", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttReceiveTopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttLogTopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttRawRopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttSendTopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttConfigTopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttOtaTopic", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttBroker", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttBrokerPort", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttUser", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttPassword", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("mqttRetain", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("rfReceiverPin", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("rfTransmitterPin", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("rfEchoMessages", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("rfProtocols", protocolInputField, protocolApply, protocolGet, "HELP"),
-    new ConfigItem("otaPassword", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("otaUrl", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("serialLogLevel", logLevelInputFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("webLogLevel", logLevelInputFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("configUser", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("configPassword", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("syslogLevel", logLevelInputFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("syslogHost", inputFieldFactory, inputApply, inputGet, "HELP"),
-    new ConfigItem("syslogPort", inputFieldFactory, inputApply, inputGet, "HELP")
+    new GroupItem("General Config", legendFactory),
+    new ConfigItem("deviceName", inputFieldFactory, inputApply, inputGet, "The general name of the device"),
+    new ConfigItem("mdnsName", inputFieldFactory, inputApply, inputGet, "The name the device shall announce via mdns"),
+    new ConfigItem("configUser", inputFieldFactory, inputApply, inputGet, "The admin username for the web UI"),
+    new ConfigItem("configPassword", inputFieldFactory, inputApply, inputGet, "The admin password for the web UI"),
+    new ConfigItem("otaPassword", inputFieldFactory, inputApply, inputGet, "The password for pull OTA"),
+    new ConfigItem("otaUrl", inputFieldFactory, inputApply, inputGet, "The OTA pull URL"),
```

Comment:
This should not be exposed, as it belongs to the mqtt-ota protocol. It even should not be in Settings.